### PR TITLE
Merge large bucket with distributed docs

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
Merge large bucket with distributed docs

## Motivation and Context
@abperiasamy asked the large-bucket doc to be deprecated and merge with the distributed doc. Also updated the image. 

## Regression
No

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :x: [more...](#c0131f23-2633-4b3f-a898-1b4425e55d41) |
| mint-large-bucket.sh | :x: [more...](#bbc5c862-a2b4-4494-8b10-9e8c225ac1f9) |
| mint-dist-xl.sh | :x: [more...](#2e38eecc-b49c-403f-ae47-64858f667889) |
<a name="c0131f23-2633-4b3f-a898-1b4425e55d41"></a>
### 7761-24466d9/mint-compression-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:30569
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 979de6d0b7d8:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-duu50t2yb5znj1",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 360014,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 15A626BBBD0AB50F, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```
<a name="2e38eecc-b49c-403f-ae47-64858f667889"></a>
### 7761-24466d9/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:31499
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d72f700a5484:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 3 seconds
(5/14) Running awscli tests ... done in 2 minutes and 0 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 58 seconds
(8/14) Running minio-dotnet tests ... done in 39 seconds
(9/14) Running minio-go tests ... done in 50 seconds
(10/14) Running minio-java tests ... FAILED in 38 seconds
{
  "name": "minio-java",
  "function": "listIncompleteUploads(String bucketName, String prefix)",
  "duration": 1198,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code=SlowDown, message=Please reduce your request, bucketName=minio-java-test-3cloraf, objectName=minio-java-test-1lg0m02, resource=/minio-java-test-3cloraf/minio-java-test-1lg0m02, requestId=15A626B9B0722843, hostId=27f9fde2-f914-4fad-9a4d-06154ba04e33)\nrequest={method=DELETE, url=http://72.28.97.57:31499/minio-java-test-3cloraf/minio-java-test-1lg0m02?uploadId=f0170ddf-684e-4164-a482-4c71e64f4677, headers=Host: 72.28.97.57:31499\nUser-Agent: MinIO (amd64; amd64) minio-java/dev\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20190608T065459Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20190608/us-east-1/s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO/DEVELOPMENT.2019-06-08T06-36-16Z\nVary: Origin\nX-Amz-Request-Id: 15A626B9B0722843\nX-Minio-Deployment-Id: 27f9fde2-f914-4fad-9a4d-06154ba04e33\nX-Xss-Protection: 1; mode=block\nDate: Sat, 08 Jun 2019 06:55:00 GMT\nTransfer-Encoding: chunked\n}\n >>> [io.minio.MinioClient.executeReq(MinioClient.java:1204), io.minio.MinioClient.execute(MinioClient.java:1070), io.minio.MinioClient.executeDelete(MinioClient.java:1347), io.minio.MinioClient.abortMultipartUpload(MinioClient.java:5149), io.minio.MinioClient.putObject(MinioClient.java:4337), io.minio.MinioClient.putObject(MinioClient.java:4183), FunctionalTest.listIncompleteUploads_test2(FunctionalTest.java:1639), FunctionalTest.runTests(FunctionalTest.java:2989), FunctionalTest.main(FunctionalTest.java:3118)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="bbc5c862-a2b4-4494-8b10-9e8c225ac1f9"></a>
### 7761-24466d9/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:32683
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 08a168de5eb3:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 47 seconds
(4/14) Running aws-sdk-ruby tests ... done in 14 seconds
(5/14) Running awscli tests ... done in 2 minutes and 3 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... FAILED in 2 minutes and 16 seconds
{
  "name": "mc",
  "duration": "94914",
  "function": "test_mirror_list_objects_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-939 --quiet --no-color mirror --storage-class REDUCED_REDUNDANCY /mint/data myminio/mc-test-bucket-651  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-651/datafile-1-MB`\n`/mint/data/datafile-0-b` -> `myminio/mc-test-bucket-651/datafile-0-b`\n`/mint/data/datafile-1-b` -> `myminio/mc-test-bucket-651/datafile-1-b`\n`/mint/data/datafile-1-kB` -> `myminio/mc-test-bucket-651/datafile-1-kB`\n`/mint/data/datafile-1.03-MB` -> `myminio/mc-test-bucket-651/datafile-1.03-MB`\n`/mint/data/datafile-10-MB` -> `myminio/mc-test-bucket-651/datafile-10-MB`\n`/mint/data/datafile-10-kB` -> `myminio/mc-test-bucket-651/datafile-10-kB`\n`/mint/data/datafile-11-MB` -> `myminio/mc-test-bucket-651/datafile-11-MB`\n`/mint/data/datafile-100-kB` -> `myminio/mc-test-bucket-651/datafile-100-kB`\n`/mint/data/datafile-129-MB` -> `myminio/mc-test-bucket-651/datafile-129-MB`\n`/mint/data/datafile-33-kB` -> `myminio/mc-test-bucket-651/datafile-33-kB`\n`/mint/data/datafile-5-MB` -> `myminio/mc-test-bucket-651/datafile-5-MB`\n`/mint/data/datafile-5243880-b` -> `myminio/mc-test-bucket-651/datafile-5243880-b`\n`/mint/data/datafile-65-MB` -> `myminio/mc-test-bucket-651/datafile-65-MB`\n`/mint/data/datafile-6-MB` -> `myminio/mc-test-bucket-651/datafile-6-MB`\nmc: <ERROR> Failed to copy `/mint/data/datafile-11-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-10-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-129-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-65-MB`. Please reduce your request\nTotal: 233.17 MiB, Transferred: 546.33 MiB, Speed: 5.76 MiB/s"
}

Executed 6 out of 14 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#d62c89b6-559b-49cb-8c4e-bbc1db4b4145) |
| mint-dist-xl.sh | :x: [more...](#0c952adb-0894-46a0-ba9b-58772924a700) |
<a name="0c952adb-0894-46a0-ba9b-58772924a700"></a>
### 7761-7f66b03/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:30963
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 223214d0bddf:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 44 seconds
(4/14) Running aws-sdk-ruby tests ... done in 3 seconds
(5/14) Running awscli tests ... done in 2 minutes and 3 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 1 minutes and 6 seconds
(8/14) Running minio-dotnet tests ... done in 42 seconds
(9/14) Running minio-go tests ... done in 1 minutes and 10 seconds
(10/14) Running minio-java tests ... FAILED in 1 minutes and 43 seconds
{
  "name": "minio-java",
  "function": "presignedGetObject(String bucketName, String objectName)",
  "duration": 93,
  "status": "FAIL",
  "error": "java.lang.Exception: failed to read object. Response: Response{protocol=http/1.1, code=503, message=Service Unavailable, url=http://72.28.97.56:30963/minio-java-test-e51hbs/minio-java-test-vd0v84?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=minio%2F20190608%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20190608T072048Z&X-Amz-Expires=604800&X-Amz-SignedHeaders=host&X-Amz-Signature=ebfa5b8b7535f215c0396a96f67079f1363f993844dcb26bfcdaa705469e2cf9}, Response body: <?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>SlowDown</Code><Message>Please reduce your request</Message><Key>minio-java-test-vd0v84</Key><BucketName>minio-java-test-e51hbs</BucketName><Resource>/minio-java-test-e51hbs/minio-java-test-vd0v84</Resource><RequestId>15A62822272D97A8</RequestId><HostId>acb49b16-c107-4835-97ed-759d6cb7a8bd</HostId></Error> >>> [FunctionalTest.readObject(FunctionalTest.java:201), FunctionalTest.presignedGetObject_test1(FunctionalTest.java:1767), FunctionalTest.runTests(FunctionalTest.java:2994), FunctionalTest.main(FunctionalTest.java:3118)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="d62c89b6-559b-49cb-8c4e-bbc1db4b4145"></a>
### 7761-7f66b03/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.52:32013
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 7419c7fc4854:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 2 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 48 seconds
(4/14) Running aws-sdk-ruby tests ... done in 13 seconds
(5/14) Running awscli tests ... done in 2 minutes and 13 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... FAILED in 1 minutes and 17 seconds
{
  "name": "mc",
  "duration": "73948",
  "function": "test_put_object_with_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-7561 --quiet --no-color cp --storage-class REDUCED_REDUNDANCY /mint/data/datafile-1-MB myminio/mc-test-bucket-31933/mc-test-object-9679  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-31933/mc-test-object-9679`\nmc: <ERROR> Failed to copy `/mint/data/datafile-1-MB`. Please reduce your request\nmc: <ERROR> Session safely terminated. To resume session `mc session resume RxSQSkYJ`"
}

Executed 6 out of 14 tests successfully.
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#281fd4d5-1c7e-450e-93be-d96a73b380ed) |
| mint-dist-xl.sh | :x: [more...](#f30398ec-2e4d-4722-9441-81cf5da25ad9) |
<a name="f30398ec-2e4d-4722-9441-81cf5da25ad9"></a>
### 7761-bc983c4/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:31933
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 76ebc90d3fdb:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 3 seconds
(5/14) Running awscli tests ... done in 1 minutes and 57 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 57 seconds
(8/14) Running minio-dotnet tests ... done in 42 seconds
(9/14) Running minio-go tests ... done in 55 seconds
(10/14) Running minio-java tests ... FAILED in 33 seconds
{
  "name": "minio-java",
  "function": "presignedGetObject(String bucketName, String objectName, Integer expires, Map<String, String> reqParams)",
  "duration": 32,
  "status": "FAIL",
  "error": "java.lang.Exception: failed to read object. Response: Response{protocol=http/1.1, code=503, message=Service Unavailable, url=http://72.28.97.57:31933/minio-java-test-1q8qede/minio-java-test-33hcq49?response-content-type=application%2Fjson&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=minio%2F20190608%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20190608T231949Z&X-Amz-Expires=3600&X-Amz-SignedHeaders=host&X-Amz-Signature=2c50d21e4a2c782b2c50fc90570c15392a523dcd6872390d9511f2ffb45c9573}, Response body: <?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>SlowDown</Code><Message>Please reduce your request</Message><Key>minio-java-test-33hcq49</Key><BucketName>minio-java-test-1q8qede</BucketName><Resource>/minio-java-test-1q8qede/minio-java-test-33hcq49</Resource><RequestId>15A65C77A5698A31</RequestId><HostId>1ed7acbe-c781-443c-874b-e2afb13b839f</HostId></Error> >>> [FunctionalTest.readObject(FunctionalTest.java:201), FunctionalTest.presignedGetObject_test3(FunctionalTest.java:1842), FunctionalTest.runTests(FunctionalTest.java:2996), FunctionalTest.main(FunctionalTest.java:3118)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="281fd4d5-1c7e-450e-93be-d96a73b380ed"></a>
### 7761-bc983c4/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.53:32578
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp dafd12dd1ee2:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 49 seconds
(4/14) Running aws-sdk-ruby tests ... done in 13 seconds
(5/14) Running awscli tests ... done in 2 minutes and 13 seconds
(6/14) Running healthcheck tests ... done in 1 seconds
(7/14) Running mc tests ... FAILED in 2 minutes and 34 seconds
{
  "name": "mc",
  "duration": "114008",
  "function": "test_mirror_list_objects_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-17090 --quiet --no-color mirror --storage-class REDUCED_REDUNDANCY /mint/data myminio/mc-test-bucket-30638  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-30638/datafile-1-MB`\n`/mint/data/datafile-0-b` -> `myminio/mc-test-bucket-30638/datafile-0-b`\n`/mint/data/datafile-1-kB` -> `myminio/mc-test-bucket-30638/datafile-1-kB`\n`/mint/data/datafile-1-b` -> `myminio/mc-test-bucket-30638/datafile-1-b`\n`/mint/data/datafile-10-MB` -> `myminio/mc-test-bucket-30638/datafile-10-MB`\n`/mint/data/datafile-1.03-MB` -> `myminio/mc-test-bucket-30638/datafile-1.03-MB`\n`/mint/data/datafile-100-kB` -> `myminio/mc-test-bucket-30638/datafile-100-kB`\n`/mint/data/datafile-10-kB` -> `myminio/mc-test-bucket-30638/datafile-10-kB`\n`/mint/data/datafile-11-MB` -> `myminio/mc-test-bucket-30638/datafile-11-MB`\n`/mint/data/datafile-129-MB` -> `myminio/mc-test-bucket-30638/datafile-129-MB`\n`/mint/data/datafile-5-MB` -> `myminio/mc-test-bucket-30638/datafile-5-MB`\n`/mint/data/datafile-33-kB` -> `myminio/mc-test-bucket-30638/datafile-33-kB`\n`/mint/data/datafile-5243880-b` -> `myminio/mc-test-bucket-30638/datafile-5243880-b`\n`/mint/data/datafile-6-MB` -> `myminio/mc-test-bucket-30638/datafile-6-MB`\n`/mint/data/datafile-65-MB` -> `myminio/mc-test-bucket-30638/datafile-65-MB`\nmc: <ERROR> Failed to copy `/mint/data/datafile-65-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-129-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-11-MB`. Please reduce your request\nTotal: 233.17 MiB, Transferred: 592.55 MiB, Speed: 5.20 MiB/s"
}

Executed 6 out of 14 tests successfully.
```

File: docs/distributed/README.md [hunk 1]
```diff
@@ -14,23 +14,23 @@ Distributed MinIO provides protection against multiple node/drive failures and [
...
 
-For example, an 8-node distributed MinIO setup with 1 disk per node would continue serving files, even if up to 4 disks are offline. But, you'll need at least 5 disks online to create new objects.
+For example, an 16-node distributed MinIO setup with 16 disks per node would continue serving files, even if up to 8 servers are offline. But, you'll need at least 9 servers online to create new objects.
 
 ### Limits
 
-As with MinIO in stand-alone mode, distributed MinIO has a per tenant limit of minimum 2 and maximum 32 servers. There are no limits on number of disks shared across these servers. If you need a multiple tenant setup, you can easily spin up multiple MinIO instances managed by orchestration tools like Kubernetes.
+As with MinIO in stand-alone mode, distributed MinIO has a per tenant limit of minimum of 2 and maximum of 32 servers. There are no limits on number of disks across these servers. If you need a multiple tenant setup, you can easily spin up multiple MinIO instances managed by orchestration tools like Kubernetes, Docker Swarm etc.
```
Comment (kannappanr) [inline docs/distributed/README.md:21]: Do we still have a 32 server limit?

File: docs/distributed/README.md [hunk 2]
```diff
@@ -43,25 +43,25 @@ To start a distributed MinIO instance, you just need to pass drive locations as
 *Note*
 
 - All the nodes running distributed MinIO need to have same access key and secret key for the nodes to connect. To achieve this, it is **mandatory** to export access key and secret key as environment variables, `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY`, on all the nodes before executing MinIO server command.
-- All the nodes running distributed MinIO need to be in a homogeneous environment, i.e. same operating system, same number of disks and same interconnects.
-- `MINIO_DOMAIN` environment variable should be defined and exported if domain is needed to be set.
-- MinIO distributed mode requires fresh directories. If required, the drives can be shared with other applications. You can do this by using a sub-directory exclusive to minio. For example, if you have mounted your volume under `/export`, pass `/export/data` as arguments to MinIO server.
+- All the nodes running distributed MinIO setup are recommended to be in homogeneous environment, i.e. same operating system, same number of disks and same network interconnects.
```
Comment (kannappanr) [inline docs/distributed/README.md:46]: Should we also say that they should be same type of device like `all SSDs, HDDs or NVMes`

File: docs/distributed/README.md (hunk 1 above)
Comment (harshavardhana) [reply docs/distributed/README.md:21]: Yes we do until PR is merged #7652

File: docs/distributed/README.md (hunk 2 above)
Comment (harshavardhana) [reply docs/distributed/README.md:46]: Suggest a change, I can accept it here in the PR.

File: docs/distributed/README.md [hunk 3]
```diff
@@ -14,23 +14,23 @@ Distributed MinIO provides protection against multiple node/drive failures and [
 
 A stand-alone MinIO server would go down if the server hosting the disks goes offline. In contrast, a distributed MinIO setup with _n_ disks will have your data safe as long as _n/2_ or more disks are online. You'll need a minimum of _(n/2 + 1)_ [Quorum](https://github.com/minio/dsync#lock-process) disks to create new objects though.
 
-For example, an 8-node distributed MinIO setup with 1 disk per node would continue serving files, even if up to 4 disks are offline. But, you'll need at least 5 disks online to create new objects.
+For example, an 16-node distributed MinIO setup with 16 disks per node would continue serving files, even if up to 8 servers are offline. But, you'll need at least 9 servers online to create new objects.
```
Comment (deekoder) [inline docs/distributed/README.md:17]: **a** 16 node i think.

File: docs/distributed/README.md (hunk 1 above)
Comment (deekoder) [inline docs/distributed/README.md:21]: I think we should write it without the limit such that when 7652 is merged, it will make sense.

**Review by deekoder — APPROVED** [review]
Great Idea to merge it. I would even merge federation inside distributed.

File: docs/distributed/README.md (hunk 1 above)
Comment (harshavardhana) [reply docs/distributed/README.md:21]: Yes it will be done once that PR is merged.

File: docs/distributed/README.md (hunk 3 above)
Comment (harshavardhana) [reply docs/distributed/README.md:17]: Suggest a change using Github suggestion feature,  I can automatically merge it to this PR.

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-fs.sh | :x: [more...](#daf77639-857d-4b64-b600-ee7e1eccf5b0) |
| mint-dist-xl.sh | :x: [more...](#6c9b916d-e753-4310-8e9d-b13d23359ddc) |
<a name="6c9b916d-e753-4310-8e9d-b13d23359ddc"></a>
### 7761-cdcc049/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:30111
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 70a3e9445c7e:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 44 seconds
(4/14) Running aws-sdk-ruby tests ... done in 3 seconds
(5/14) Running awscli tests ... done in 59 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 33 seconds
(8/14) Running minio-dotnet tests ... done in 33 seconds
(9/14) Running minio-go tests ... done in 1 minutes and 31 seconds
(10/14) Running minio-java tests ... FAILED in 1 minutes and 17 seconds
{
  "name": "minio-java",
  "function": "removeIncompleteUpload(String bucketName, String objectName)",
  "duration": 1003,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code=SlowDown, message=Please reduce your request, bucketName=minio-java-test-3ma7hko, objectName=minio-java-test-vo5r3n, resource=/minio-java-test-3ma7hko/minio-java-test-vo5r3n, requestId=15A6E8D3B3D90F8F, hostId=4270f753-0813-4f31-91df-1123503442f0)\nrequest={method=DELETE, url=http://72.28.97.61:30111/minio-java-test-3ma7hko/minio-java-test-vo5r3n?uploadId=154aeed2-b9b8-411d-934f-0140a2237f34, headers=Host: 72.28.97.61:30111\nUser-Agent: MinIO (amd64; amd64) minio-java/dev\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20190610T181156Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20190610/us-east-1/s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO/DEVELOPMENT.2019-06-10T17-53-16Z\nVary: Origin\nX-Amz-Request-Id: 15A6E8D3B3D90F8F\nX-Minio-Deployment-Id: 4270f753-0813-4f31-91df-1123503442f0\nX-Xss-Protection: 1; mode=block\nDate: Mon, 10 Jun 2019 18:11:57 GMT\nTransfer-Encoding: chunked\n}\n >>> [io.minio.MinioClient.executeReq(MinioClient.java:1204), io.minio.MinioClient.execute(MinioClient.java:1070), io.minio.MinioClient.executeDelete(MinioClient.java:1347), io.minio.MinioClient.abortMultipartUpload(MinioClient.java:5149), io.minio.MinioClient.putObject(MinioClient.java:4337), io.minio.MinioClient.putObject(MinioClient.java:4183), FunctionalTest.removeIncompleteUploads_test(FunctionalTest.java:1719), FunctionalTest.runTests(FunctionalTest.java:2992), FunctionalTest.main(FunctionalTest.java:3118)]"
}

Executed 9 out of 14 tests successfully.
```
<a name="daf77639-857d-4b64-b600-ee7e1eccf5b0"></a>
### 7761-cdcc049/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:31174
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 99926a97a175:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 3 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-qeshmbemij1bt4",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 3071,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:31174/aws-sdk-go-test-qeshmbemij1bt4: dial tcp 72.28.97.57:31174: connect: no route to host",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Review by kannappanr — APPROVED** [review]
LGTM

**Pull request merged**